ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/perception/RDXContinuousHikingPanel.java

```diff
@@ -118,59 +114,16 @@ public RDXContinuousHikingPanel(ROS2Helper ros2Helper,
-                                                                       syncedRobot.getRobotModel().getSimpleRobotName()), message ->
-                                      {
-                                         terrainPlanningDebugger.reset();
-                                      });
-
-
-      LogTools.info("Continuous Walking Parameters Save File " + continuousHikingParameters.findSaveFileDirectory().toString());
-      continuousPlanningParametersTuner.create(continuousHikingParameters, false);
-   }
-
-   public RDXContinuousHikingPanel(ROS2Helper ros2Helper, DRCRobotModel robotModel, MonteCarloFootstepPlannerParameters monteCarloPlannerParameters)
-   {
-      super("Continuous Hiking");
-      setRenderMethod(this::renderImGuiWidgets);
-
-      SegmentDependentList<RobotSide, ArrayList<Point2D>> groundContactPoints = robotModel.getContactPointParameters().getControllerFootGroundContactPoints();
-      SideDependentList<ConvexPolygon2D> defaultContactPoints = new SideDependentList<>();
-      for (RobotSide robotSide : RobotSide.values)
-      {
-         ConvexPolygon2D defaultFoothold = new ConvexPolygon2D();
-         groundContactPoints.get(robotSide).forEach(defaultFoothold::addVertex);
-         defaultFoothold.update();
-         defaultContactPoints.put(robotSide, defaultFoothold);
-      }
-
-      stancePoseCalculator = new StancePoseCalculator(0.5f, 0.5f, 0.1f, defaultContactPoints);
-
-      this.ros2Helper = ros2Helper;
-      this.swingPlannerParameters = robotModel.getSwingPlannerParameters();
-      this.swingTrajectoryParameters = robotModel.getWalkingControllerParameters().getSwingTrajectoryParameters();
-      this.commandPublisher = ros2Helper.getROS2NodeInterface().createPublisher(ContinuousWalkingAPI.CONTINUOUS_WALKING_COMMAND);
-      this.stancePoseSelectionPanel = new RDXStancePoseSelectionPanel(stancePoseCalculator);
-      this.terrainPlanningDebugger = new RDXTerrainPlanningDebugger(ros2Helper,
-                                                                    monteCarloPlannerParameters,
-                                                                    robotModel.getContactPointParameters().getControllerFootGroundContactPoints());
-
-      ros2Helper.subscribeViaCallback(ContinuousWalkingAPI.START_AND_GOAL_FOOTSTEPS, this::onStartAndGoalPosesReceived);
-      ros2Helper.subscribeViaCallback(ContinuousWalkingAPI.PLANNED_FOOTSTEPS, this::onPlannedFootstepsReceived);
-      ros2Helper.subscribeViaCallback(ContinuousWalkingAPI.MONTE_CARLO_FOOTSTEP_PLAN, this::onMonteCarloPlanReceived);
-      ros2Helper.subscribeViaCallback(HumanoidControllerAPI.getTopic(WalkingControllerFailureStatusMessage.class, robotModel.getSimpleRobotName()), message ->
-      {
-         terrainPlanningDebugger.reset();
-      });
+                                                               robotModel.getContactPointParameters().getControllerFootGroundContactPoints());
```

Comment:
GitHub does a terrible job but there were 2 constructors for this class and I combined them into one